src/components/HireMe/index.js

```
@@ -0,0 +1,367 @@
+import { MailIcon, PhoneIcon } from "@heroicons/react/outline";
+import emailjs from "emailjs-com";
+import React, { useRef } from 'react';
+
+export default function HireMe() {
+  const form = useRef();
+
+  const sendEmail = (e) => {
+    e.preventDefault();
+
+    emailjs.sendForm('service_f2ne3is', 'template_lqnvp9h', form.current, 'user_vVXH8VL0YnDG8Gx4iRCqj')
+    .then((result) => {
+        console.log(result.text);
+    }, (error) => {
+        console.log(error.text);
```

Same as above, we should actually show an error message to the user instead of failing silently